### ISSUE TITLE
Sharesheet link: No auth method found!

### ISSUE THREAD

**Issue by WinstonSmith73** [Dacid99/LinkDroid-for-Linkwarden#10]
I set up an account for my wife on my self-hosted Linkwarden instance. I can log her in through LinkDroid (or Brave) but when I try to save a link via sharing from Brave I get "No auth method found!" error.

Works fine for me in my account on my phone, from Brave. 

I've deleted LinkDroid cache & storage, reinstalled it, and restarted the server. 

I can save links in LinkDroid by pasting them in via gBoard.

Android 14
CalyxOS 5.12.0
Pixel 6

**Comment by Dacid99** [comment]
Thats odd, have you set up the access token in LinkDroid?
Alternatively it's sufficient to be logged in via the Linkdroid app so there is a valid cookie.
What app version are you on?

**Comment by WinstonSmith73** [comment]
v1.4.0. No access token, but successfully logged in via LinkDroid. Didn't see an option to get an access token in her LW account.

**Comment by Dacid99** [comment]
Did this also happen on an earlier version?

**Comment by WinstonSmith73** [comment]
Yes, logged-in when reopening. Don't have to login every time. Tried reverting to 1.3.0 but same result. 

This is a pretty new LW server so not much history of use with LD.

**Comment by Dacid99** [comment]
Ok thats odd. It cant be the servers fault. It works on your own phone which is also android 14, so it cant be that either. 
Does this also happen when sharing from another browser (i test with firefox) ?

**Comment by WinstonSmith73** [comment]
I just tried it with Fennec (Firefox) and it gave the same error :(

**Comment by WinstonSmith73** [comment]
I tried making a new account for her, with the same result. Then I generated an access token from desktop, entered it, and it works now! Not sure why it won't work via share without the token but works just fine otherwise.

Let me know if you want to help debug any further.

**Comment by Dacid99** [comment]
Ok it's very good that you found a solution :+1: .

There has to be something going wrong with the cookies in the app storage.

The next versions of the app will bring some improved logging in that area so we may be able to get a better grip of the problem then.
